docs/purchaseCkb.md

```diff
@@ -0,0 +1,85 @@
+:::
+
+## Problem with locks and exchanges
+
+Sometimes when you’re withdrawing from an exchange to `L1 Wallet Address` or `L1 Deposit Adderss`, the exchange could stop you and warn that your recipient’s address is invalid, or in a worse case the withdrawal just fails without a warning.
+
+The reason why your withdrawals may fail may be because the exchange you’re using does not support the `Omni-lock` and the `Deposit-lock`, the locks used on `L1 Wallet Address` and `L1 Deposit Adderss`.
+
+### Finding the reason
+
+We don't want to overcomplicate things here, so I won't explain everything about locks but share some reading material, so you can read them individually. 
+
+So, in the world of CKB L1, there are a lot of locks for different situations. For instance, there’s a [SECP256K1/blake160](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md#secp256k1blake160) lock as the default lock script to verify CKB transaction signatures. And as the default lock of CKB, it is commonly supported by exchanges and crypto wallets.
+
+But in Godwoken’s perspective, we need to support transaction signing methods from the outside world, from other popular blockchains like `Ethereum` and `Bitcoin`, and we cannot do this with only the `SECP256K1/blake160` lock. So, here we have some other locks:
+
+- [Omni-lock](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md): The lock is designed for interoperability with built-in support for verification of transaction signing methods used in Ethereum, Bitcoin and more, you can also read this blog for more details: [https://blog.cryptape.com/omnilock-a-universal-lock-that-powers-interoperability-1](https://blog.cryptape.com/omnilock-a-universal-lock-that-powers-interoperability-1).
+- [Deposit-lock](https://github.com/godwokenrises/godwoken/blob/ad898e7a9821df3e627737d32865bec85bf1a753/docs/deposit_and_withdrawal.md#deposit): The lock is for depositing assets from CKB L1 to Godwoken L2, the block producer of Godwoken will collect cells with deposit-lock and process the deposit requests in blocks.
+
+If the above content is still vague to you, perhaps think of it this way: Different locks correspond to different L1 addresses, and some exchanges just don’t reorganize locks other than the `SECP256K1/blake160` lock. Therefore `L1 Wallet Address` (Omni-lock) and `L1 Deposit Address` (Deposit-lock) may not be valid in the eyes of these exchanges.
+
+### How to solve it
+
+If you have encountered this issue, you can follow the below instructions to safely transfer your CKB to the `L1 Wallet Address` or the `L1 Deposit Address` of the target account.
+
+Since the problem we’re facing is that the exchange don’t support the type of address we tend to transfer to, the overall solution is to add an extra step in the middle of buying CKB. Let’s take `L1 Wallet Address` as an example. 
```

Comment:
```suggestion
Since the problem we’re facing is that the exchange doesn’t support the type of address we tend to transfer to, the overall solution is to add an extra step in the middle of buying CKB. Let’s take `L1 Wallet Address` as an example. 
```